### ISSUE TITLE
Child rows are not compatible with fixedHeader

### ISSUE THREAD

**Issue by in-op** [DataTables/FixedHeader#103]
If you have a table that has both a fixedHeader and expandable child rows with `.child()`, then when you expand the child row the fixedHeader doesn't seem to re-calculate the table height and will end prematurely if you scroll down.

**Comment by DataTables** [comment]
You are absolutely correct. What you need to do at the moment is call the [`fixedHeader.adjust()` method](https://datatables.net/reference/api/fixedHeader.adjust()) just after you show the child row. We should perhaps have DataTables trigger an event on child row show / hide and have FixedHeader listen for that automatically...
